pkg/pipelines/bootstrap.go

```diff
@@ -536,3 +537,22 @@ func getResourceFiles(res res.Resources) []string {
 	sort.Strings(files)
 	return files
 }
+
+//check if the file exists or not
+func CheckFileExists(fs afero.Fs, dockerConfigJSONFilename string) (string, error) {
+	if dockerConfigJSONFilename == "" {
+		return "", errors.New("failed to generate path to file: must provide --dockerconfigjson parameter")
+	}
+	authJSONPath, err := homedir.Expand(dockerConfigJSONFilename)
+	if err != nil {
+		return "", fmt.Errorf("failed to generate path to file: %v", err)
+	}
+	_, err = fs.Stat(authJSONPath)
+	if os.IsNotExist(err) {
+		return "", fmt.Errorf("Please input the correct file path to the --dockerconfigjson parameter")
```

Comment:
This should probably say that the file didn't exist and that's why we're returning this error?